### PR TITLE
validation on css

### PR DESCRIPTION
**Pull request by digomes87**

### CODE REVIEW

File: index.htm [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        </ul>
+      </article>
+
+      <article class="latest-post-red">
+        <div class="latest-post-red"></div>
+      </article>
+      <article></article>
+    </div>
+
+    <footer></footer>
```
Comment (samgaco) [inline index.htm]: The footer should contain centered links.

**Review by samgaco — CHANGES_REQUESTED** [review]
#### Hi @digomes87 and @jairjaramillo :hand: 

#### Everything is looking smooth. Just some points to cover before proceeding:

* The footer should contain centered links
* [OPTIONAL] Correct the warnings given to you by the [validator](https://validator.w3.org) when updating the html file

**Work on these changes and submit back for a code review once you are done :+1:** 

#### Regards! :muscle:

File: index.htm [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+      <div class="footer-links">
+          <a href="">Lorem</a>
+          <a href="">Ipsum</a>
+          <a href="">Dolor</a>
+          <a href="">Sit</a><br>
+          <a href="">Amet</a>
+          <a href="">Consectetur</a>
+          <a href="">Adipisicing</a>
+      </div>
+    </footer>
```
Comment (samgaco) [inline index.htm:340]: Well done :+1:

**Review by samgaco — APPROVED** [review]
#### Everything is ok now. Congratulations on the project, it is looking good :+1: You are approved :medal_sports: 

##### You may merge to the master branch and submit the project completion form :stars:

**Review by samgaco — APPROVED** [review]

**Review by jairjaramillo — APPROVED** [review]
Merging with master

**Pull request merged**